lib/sonata/tip/getUserTipInfo.ts

```diff
@@ -0,0 +1,38 @@
+import { createClient } from "@supabase/supabase-js";
+import verifySignerUUID from "../../neynar/verifySigner";
+import getChannelTipInfo from "./getChannelTipInfo";
+
+const SUPABASE_URL = process.env.SUPABASE_URL as string;
+const SUPABASE_KEY = process.env.SUPABASE_KEY as string;
+
+const supabase = createClient(SUPABASE_URL, SUPABASE_KEY);
```

Comment:
I've seen this code block `twice` in this PR
- can you please DRY your code by using a shared lib to get `supabase` instance?